### ISSUE TITLE
Authentication fails when save games is enabled in config

### ISSUE THREAD

**Issue by Matthew-Prinold** [playgameservices/play-games-plugin-for-unity#2429]
When I remove the enabledSavedGames from the config my app is able to authenticate and sign in the tester however when enabled it no longer works. I have checked the SHA1 certificates and they all match, save games is enabled in the developer console and can be seen as enabled in the API console. I am at a loss of how to fix this issue. I have checked that I have the latest relese of android SDK play services library and resources and google API for the android versions in SDK Manager, they are all upto date. I have checked and double checked the package name in both unity and the developer console and they both match. I have even built a new project and imported the cubic pilot sample, set up a new game services project and configured according to the documentation and I still get the same problem. 

Any help will be much appreciated.

**Comment by ProtoJerry** [comment]
I have almost same issue here. Without .EnableSavedGames() everything works fine (Sign-in / Leader-boards & achievements). I already tried everything I could find. Double-triple checked the auth debug and release credentials are all working fine and the Saved Games is enabled in console. Same thing with a fresh unity project and CubicPilot sample, new game service - the works.
This is what I always get when I set .EnableSavedGames() in the config:
E/TokenFragment: Setting result error status code to: 16
D/TokenPendingResult:  Calling onResult for callback: GooglePlayGames.Android.AndroidTokenClient+ResultCallbackProxy <c# proxy java object> result: Status: Status{statusCode=CANCELED, resolution=null} email: <null> id:<null> access: <null>
And the string I get from Social.localUser.Authenticate is "Authentication canceled"

The email selection and the OAuth consent shows up .. but after I allow and select email it just closes and does nothing. And I also checked permissions are given in account access on the app - "Has access to Google Drive, Google Play"

@Matthew-Prinold - Op: SIGN_IN status: ERROR_NOT_AUTHORIZED - You only get this error? This is what happens to me if I don't set or remove the credentials.

**Comment by Matthew-Prinold** [comment]
What do you mean by reset or remove your credentials?

**Comment by ProtoJerry** [comment]
Like the testers are not set right/Saved games is off in console. I have also checked my logs on the first run I get : 
 09:40:10.731 2026-2026/? I/Unity: Building GPG services, implicitly attempts silent auth
    (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)
01-27 09:40:10.741 2026-2026/? I/GamesNativeSDK: Using existing jar.
01-27 09:40:10.741 2026-2026/? I/GamesNativeSDK: Writing 1941 bytes to jar file
01-27 09:40:10.741 2026-2026/? I/GamesNativeSDK: Using existing jar.
01-27 09:40:10.741 2026-2026/? I/GamesNativeSDK: Writing 1066 bytes to jar file
01-27 09:40:10.751 2026-2026/? W/PopupManager: You have not specified a View to use as content view for popups. Falling back to the Activity content view. Note that this may not work as expected in multi-screen environments
01-27 09:40:10.761 2026-2356/? I/GamesNativeSDK: Auth operation started: SIGN IN
01-27 09:40:10.761 2026-2356/? I/GamesNativeSDK: Connecting to Google Play...
01-27 09:40:10.801 2026-2026/? V/GamesNativeSDK: Received Activity Resume Event.
01-27 09:40:10.801 2026-2026/? D/TokenFragment: onResume called
01-27 09:40:10.801 266-1456/? W/ContextImpl: Calling a method in the system process without a qualified user: android.app.ContextImpl.sendBroadcast:1123 com.android.server.InputMethodManagerService.hideCurrentInputLocked:1880 com.android.server.InputMethodManagerService.windowGainedFocus:1968 com.android.internal.view.IInputMethodManager$Stub.onTransact:221 com.android.server.InputMethodManagerService.onTransact:818 
01-27 09:40:10.831 1251-1256/? D/dalvikvm: GC_CONCURRENT freed 403K, 11% free 4064K/4528K, paused 1ms+1ms, total 7ms
01-27 09:40:10.851 167-167/? W/SurfaceFlinger: couldn't log to binary event log: overflow.
01-27 09:40:10.961 468-924/? W/GLSUser: [DeviceKeyStore] Cannot load key: Device key file not found.
01-27 09:40:10.971 468-924/? W/GLSUser: [DeviceKeyStore] Cannot load key: Device key file not found.
    (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)
01-27 09:40:11.051 2026-2041/? I/Unity:  [Play Games Plugin DLL] 01/27/19 9:40:11 +02:00 DEBUG: Invoking user callback on game thread
    (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)
01-27 09:40:11.081 2367-2367/? D/AndroidRuntime: >>>>>> AndroidRuntime START com.android.internal.os.RuntimeInit <<<<<<
01-27 09:40:11.311 2026-2026/? V/GamesNativeSDK: Play Games callback indicates connection failure.
01-27 09:40:11.311 2026-2356/? I/GamesNativeSDK: Error connecting to Google Play: code 8.
01-27 09:40:11.351 2026-2041/? I/Unity:  [Play Games Plugin DLL] 01/27/19 9:40:11 +02:00 DEBUG: Starting Auth Transition. Op: SIGN_IN status: ERROR_NOT_AUTHORIZED
    (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)
01-27 09:40:11.351 2026-2041/? I/Unity:  [Play Games Plugin DLL] 01/27/19 9:40:11 +02:00 DEBUG: AuthState == Unauthenticated calling auth callbacks with failure

Then if I try to sign in again : 
01-27 09:42:54.131 2026-2026/? E/TokenFragment: Setting result error status code to: 16
01-27 09:42:54.131 2026-2026/? D/TokenPendingResult:  Calling onResult for callback: GooglePlayGames.Android.AndroidTokenClient+ResultCallbackProxy <c# proxy java object> result: Status: Status{statusCode=CANCELED, resolution=null} email: <null> id:<null> access: <null>
01-27 09:42:54.131 2026-2026/? V/GamesNativeSDK: Received Activity Resume Event.

**Comment by Matthew-Prinold** [comment]
I have tried removing and resetting the credentials but this has no effect, I still get the same issue. I don't believe savedgames requires OAuth web app id set but I will try it anyway.

**Comment by Matthew-Prinold** [comment]
i returned the request status and it says authorization canceled even after selecting an testing email from the UI and accepting the OAuth concent.

**Comment by ProtoJerry** [comment]
> 
> 
> i returned the request status and it says authorization canceled even after selecting an testing email from the UI and accepting the OAuth concent.

Same here. The drive api generates error 404.
![image](https://user-images.githubusercontent.com/44014255/51823386-ebc72f00-22e7-11e9-90c5-ec1794220336.png)

Can anyone else confirm if the save games are working in a current build?

**Comment by ProtoJerry** [comment]
I have tested with a web credential to use the drive but I get this error:
![image](https://user-images.githubusercontent.com/44014255/51824325-a22c1380-22ea-11e9-900c-3944e5ca321e.png)
The app is published ...
![image](https://user-images.githubusercontent.com/44014255/51824276-80cb2780-22ea-11e9-8774-11f3303b2fd0.png)

**Comment by OasisPremium** [comment]
Just wanted to say that I'm having the same exact problem. As soon as .enablesavedgames() is added to the config of the builder I get many errors in log and nothing work. If I remove it, sign-in works, achievements and stuff is all good. Triple-checked the certs/signatures but I might just be doing something wrong ? But I get the exact same errors as you.

**Comment by Tiderus** [comment]
I have the same problem. But my game isn't authorize with and without save game enable. I can't authorize to game after enable save game and now I can't even authorize without save game even on outer settings and on new game package, keystore and so on!!! I use Unity 2018.3.2f1 Personal and my device is Redmi A4

**Comment by olehkuznetsov** [comment]
Check if you have linked the app with your apk  SHA1 key in Play Console, please.

**Comment by olehkuznetsov** [comment]
Have you checked if "Saved Games" option is "On" for you as on screenshot in comment above?

**Comment by Matthew-Prinold** [comment]
I refer you to the start of the thread. Yes all these settings have
checked, double checked and retested. And also yes I have added the users
to the testers and cache and memory has been cleared from the devices as
per Google's instructions

**Comment by P-JohnSilva** [comment]
I currently have the same problem, my gpgs works perfectly without "enableSavedGames()" 
I all have SHA1 certificate correct (I made two linked apps, one containing the SHA1 from google playstore apk, and one coming from unity build)
I also ticked enable save games in the google play console. I also have the service published.

**Comment by Matthew-Prinold** [comment]
@P-P-JohnSilva

Okay I just wanted to check as I havnt opted in for the app signing yet.
Yeah I have the same situation, when I remove the enablesavegames() from
between builder() and build() in the config everything works;
authentication, leaderboards, and achievements.

**Comment by olehkuznetsov** [comment]
I can't find any game with a linked package com.porkypinegames.letter_fall in Play Games Console.
May be you have only created auth entry in cloud project and have no linked it in Play Console?

**Comment by Final35** [comment]
Hi, I just wanted to pop in to say that I seem to be having the exact same issue. 

I have been trying to set up game services in my upcoming app, and I had achievements and leaderboards integrating, working and authenticating on device. However as soon as I added enableSavedGames() to the builder, it stopped authenticating. ( my app is com.finalgamestudio.lineloops )

I swapped saves games to 'on'  on the game services dashboard and published that change a few days ago, but its still not working. I have also tried reinstalling everything on a fresh device in case something had been cached wrong.

To make sure the issue isn't related to certs I am having googlepay manage my signing & am uploading each of my test APKs, waiting for them to process and then downloading them from the playstore onto a test device and clean emulator. 

I'm not great at deciphering the android log files, but based on the API and Services dashboard, I think the problem is to do with the drive API, as it is getting 100% errors, all of which are 404 errors on drive.apps.get :

![image](https://user-images.githubusercontent.com/47339044/52243744-891af800-28d2-11e9-97ff-3c1abd738dd2.png)

Is there anything extra that needs to be set up in relation to the drive API for saved games to work?

**Comment by ProtoJerry** [comment]
I don't think the problem is from the oauth sha1 because if that is not set up right the sign-in, leaderboards, achievements, etc won't work. If the saved games tick is not enabled in the console it gives a specific error.
@olehkuznetsov can you confirm that the enableSavedGames() works in a newly created project? I would appreciate it if we knew that it works for someone. Thanks

**Comment by olehkuznetsov** [comment]
Yes, it seems there is a bug with newly created projects. Investigating it.

**Comment by olehkuznetsov** [comment]
Final35, I'm still not sure why I can't find your project by package name. Can you provide you app_id please?

**Comment by Final35** [comment]
> @Final35, I'm still not sure why I can't find your project by package name. Can you provide this project app_id, please?

Sorry - that's my fault, I left out the platform identifier I am using ( I release on multiple platorms ). The full package name on googleplay is: 'com.finalgamestudio.lineloops.android.googleplay' . The appID is 434431536870.

**Comment by sskynomad** [comment]
I have the exact same problem, and here I thought I was doing something wrong. I will back read everything.

**Comment by superswierk** [comment]
Hi, I have the same problem with only newly created game services. Is this related to Drive Android API depreciation? Apparently Game Save mechanism uses DriveApi v2 and drive.apps.get return 404 for new game services projects. It looks like the bug is inside GoogleSignInClient or its wrong usage.

https://developers.google.com/drive/android/deprecation

**Comment by andrewpmoore** [comment]
Just here to say I'm getting the same. Thought it was my SHA1 keys because it's a new project and I'd enabled enablesavedgame from the outset, so sign-in never worked.

Once I removed this, the signin and leaderboards/achievements work fine.

**Comment by smile616** [comment]
Could you please try disable and enable the Drive API via Cloud Platform console, i.e. https://console.cloud.google.com/apis/api/drive.googleapis.com/overview 
and let us know if that resolved you issue.
Thanks!

**Comment by Final35** [comment]
> Hi, I have the same problem with only newly created game services. Is this related to Drive Android API depreciation? Apparently Game Save mechanism uses DriveApi v2 and drive.apps.get return 404 for new game services projects. It looks like the bug is inside GoogleSignInClient or its wrong usage. Or maybe some issue in google web services.
> 
> https://developers.google.com/drive/android/deprecation

I have tried that now (on a fresh install), and there was no change

**Comment by Matthew-Prinold** [comment]
@smile616 @Final35 

I have also tried to no success. 

@olehkuznetsov I have even published my game on an internal track with the linked api and still no good. can you still not see my app id or project name. 

has anyone had any look getting things working yet or has there been any developments on the bug fix yet?

**Comment by Matthew-Prinold** [comment]
to add some new info it all happened when I upgraded to the latest release, previous releases worked fine. but as I wanted to include firebase I needed the latest release. That being said I have tried a new project including your examples and they don't work

**Comment by ProtoJerry** [comment]
I have also tried creating a new project and letting the console set up credentials same problem. I tried enabling disabling the drive api / remove and recreate the credentials / added all the scopes I could without the need for verification on the consent screen. No-change. 
@Matthew-Prinold Can you recall witch version was working? Also unity version?

**Comment by olehkuznetsov** [comment]
@[Matthew-Prinold](/Matthew-Prinold) Yes, I was able to find your game in our database. 
I don't think it is connected to unity/plugin version. Seems as a console/cloud configuration issue. Currently it looks like as 'drive' can't find your app_id their side.

**Comment by olehkuznetsov** [comment]
It seems only recently created cloud projects are affected as we enabled saved games on one of long time ago created projects and it works fine.

**Comment by superswierk** [comment]
Yes. I had one old game services and I switched to it (re linked it with my app) and it started to work - so only newly created game services (projects) have this problem.
Problem occurs on plugin 0.9.50 and 0.9.60.

**Comment by voxelbusters** [comment]
Even we can confirm the issue is with newly created projects where saved games is enabled. We don't use this plugin but our plugin implementation facing the same issue. This means its not specific to the plugin but with the play services libraries.

Any one filed an issue?

**Comment by olehkuznetsov** [comment]
>Any one filed an issue?
Ye

**Comment by superswierk** [comment]
maybe its related to this issue
https://issuetracker.google.com/issues/121345407

**Comment by jeffkingdev** [comment]
Has this been resolved yet?  I'm seeing the same behavior with my app -- login failed when save games are enabled.  Logcat is showing 404 reason: [notFound] location:[app].  Save games off, works fine and logs me in.  Also shows 404 for 'https://www.googleapis.com/drive/v2beta/apps/self?prettyPrint=false&fields=id'

Is there a solution or workaround for this?  Do I need to install an older play-games-plugin version?

Or is there currently no fix and we can no longer offer up cloud saves to our customers?

**Comment by olehkuznetsov** [comment]
We are investigating this issue. 
This is not related to play-games-plugin version. Java and C++ PGS SDK are affected as well.

**Comment by jeffkingdev** [comment]
Thanks @olehkuznetsov 

I've used the cloud saves in the past and it's a popular feature.  I'm hoping there's a quick fix so others aren't wondering what's going on either!  Thanks for looking into this!

**Comment by voxelbusters** [comment]
> It looks like its a problem inside google play services sdk - maybe its related to this issue
> https://issuetracker.google.com/issues/121345407

This issue doesn't seem to point to the exact problem so, we tried filing an issue in google bug report but they asked us to report to google play support org and closed the issue.

Later we filed over [here](https://github.com/playgameservices/android-basic-samples/issues/289) and there is no response from them yet, even though the issue being severe.

**Comment by Salranax** [comment]
I have the same issue. Tried every solution that mentioned here and still have it. We are gonna publish the app soon and waiting this issue to get resolved. Please update this post.

**Comment by olehkuznetsov** [comment]
This is a high priority issue. We are waiting for the fix from a drive team. They already found a reason and are working on a solution. 
Sorry, as I have no estimated time for this fix, yet.

**Comment by VaraderoCuba** [comment]
"games_lite", API ....?

I have spent much time to figure this, callback error of Authenticate, out myself in stupida way such like, finding out perfect golden combination of play-games-plugin and GoogleMobileads, each every single versions, even I was wonder there is no perfect one.
Want to forget tons of import then removing..past many days..

Because of this issue, I've penta-checked Sha-1, OAuth client ID and such things if there are some misspelled on them....   :(
Not misspelled word, but I could find remarkable difference between further project and recent project which are no 'games_lite' can be chosen at add scop in APIs-OAuth menu.
I would guess plugin is still calling 'games_lite' maybe its direct way, even if you cannot.

Just project releasing stopped cuz of this issue, love to see new patch for it or changed APIs for savedgame enabled issue... if someone resolved even temporary pls share the way, here!!!! T^T

**Comment by voxelbusters** [comment]
> This is a high priority issue. We are waiting for the fix from a drive team. They already found a reason and are working on a solution.
> Sorry, as I have no estimated time for this fix, yet.

Can you please share the bug report id to track the issue?

**Comment by Salranax** [comment]
@olehkuznetsov Can you please notify us about the fix. We are waiting this issue get fixed to publish our app and still waiting.

**Comment by SCBbestof** [comment]
@olehkuznetsov I got a load of information and I'll leave it here just in case::

I struggled for a couple of days to make cloud saves work for a new game I'm working on. Everything was setup perfectly and worked with no issues unless cloud saves were enabled. 

So, I gave up on my game today and started following the codelabs tutorial (with the Lollygagger app). I thought since that was an official tutorial it will work, but it didn't. It had the exact same issue. Enable Cloud Saves results in authentication failures. Here are some logs too:
```
I/GamesNativeSDK(31583): Auth operation started: SIGN IN                                                                       
I/GamesNativeSDK(31583): Connecting to Google Play...                                                                          
I/Unity   (31583): (Lollygagger) Sign-in failed...                                                                             
I/Unity   (31583):                                                                                                             
I/Unity   (31583): (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)                                                      
I/Unity   (31583):                                                                                                             
I/Unity   (31583):  [Play Games Plugin DLL] 02/20/19 19:04:19 +02:00 DEBUG: Invoking user callback on game thread                                    
I/Unity   (31583): (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)                     
I/Unity   (31583):  [Play Games Plugin DLL] 02/20/19 19:04:30 +02:00 DEBUG: Starting Auth Transition. Op: SIGN_IN status: ERRO 
R_TIMEOUT                                                                                                                     
I/Unity   (31583): (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)                                                         
I/Unity   (31583):  [Play Games Plugin DLL] 02/20/19 19:04:30 +02:00 DEBUG: AuthState == Unauthenticated calling auth callback 
s with failure                                                                                                                    
I/Unity   (31583): (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)                                                                         
I/Unity   (31583): Starting Auth with token client.                                                                            
I/Unity   (31583): (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)                     
D/TokenFragment(31583): Fragment exists.. calling processRequests                                                              
D/TokenFragment(31583): Building client for: 41900e88 (a:false e:false i:false wc:  f: false)                                  
D/TokenFragment(31583): signIn                                                                                                 
D/TokenFragment(31583): signInClient.silentSignIn                                                                              
D/TokenFragment(31583): Done with processRequest, result is pending.                                                           
D/TokenFragment(31583): silentSignIn.onFailure                                                                                   
D/OpenGLRenderer(31583): Enabling debug mode 0                                                                                    
V/GamesNativeSDK(31583): Play Games callback indicates connection failure.                                                     
E/GamesNativeSDK(31583): Unexpected response: connection failed.                                                               
E/TokenFragment(31583): Setting result error status code to: 16                                                                
D/TokenPendingResult(31583):  Calling onResult for callback: ResultCallbackProxy result: Status: Status{statusCode=CANCELED, r 
esolution=null} email: <null> id:<null> access: <null>                                                                         
V/GamesNativeSDK(31583): Received Activity Resume Event.                                                                       
D/TokenFragment(31583): onResume called                                                                                                                        
I/Unity   (31583):  [Play Games Plugin DLL] 02/20/19 19:06:02 +02:00 DEBUG: Application is pausing, which disconnects the RTMP 
  client.  Leaving room.                                                                                                    
I/Unity   (31583): (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)                                                                                 
I/Unity   (31583): (Lollygagger) Sign-in failed...                                                                
I/Unity   (31583): (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)                                             
I/Unity   (31583):  [Play Games Plugin DLL] 02/20/19 19:06:02 +02:00 DEBUG: Invoking user callback on game thread                                                                                          
I/Unity   (31583): (Filename: ./Runtime/Export/Debug.bindings.h Line: 45)                                                                                   
```

Drive API shows a 100% failure rate, all being 4xx errors.

Thanks for looking into this! 😃

**Comment by VergilUa** [comment]
Same for me, using different plugin / libs. Seems like Drive API failure rate is 100% on newer projects.
It's crazy how this went pass by Google QA.

**Comment by rickyfreddy** [comment]
Having the same issue. Removing "EnableSavedGames" in the PlayGamesClientConfiguration building code made sign in work, but I kinda need cloud saving to work to publish the app. Please Google fix this!

**Comment by kreesty** [comment]
I'm having this issue as well.

**Comment by Peariforme** [comment]
Same problem here. Just took me all the day to find this post and understand it is not my fault !

**Comment by sptkjs** [comment]
@olehkuznetsov Can you please notify me about the fix. I'm waiting for bug fixed.

**Comment by Blackgordian** [comment]
After days finally i discover its a bug.
Come on guys

**Comment by jeffkingdev** [comment]
Any updates on this issue?  ETA?

Thanks!

**Comment by dalight13** [comment]
Go comment on that google issue as it's coming from their side : https://issuetracker.google.com/issues/121345407

Also the priority should be changed as P4 won't be fixed anytime soon...

**Comment by olehkuznetsov** [comment]
I think this issue https://issuetracker.google.com/issues/121345407 is a different one.
We have another internal one with P1 priority. I see an active progress on it.

**Comment by voxelbusters** [comment]
We opened an issue over here (https://issuetracker.google.com/issues/124103727) but they closed saying won’t fix and redirected us to github bug reporting.

**Comment by olehkuznetsov** [comment]
Current ETA is the middle of next week.

**Comment by kaninhot004** [comment]
> Current ETA is the middle of next week.

I hope it's worked soon.

**Comment by kreesty** [comment]
Any update on this?

**Comment by kaninhot004** [comment]
"ETA is the middle of next week."

So in 2-3 day left, I think so.

**Comment by wooyaggo** [comment]
Is this sovled? 
0.9.62 is released a few days ago.
But I have save issue.

**Comment by gAmeshhill** [comment]
I too have the exact same issue
Hoping to receive the fix soon

**Comment by voxelbusters** [comment]
They seem to update the status as Won't Fix !!!

> https://issuetracker.google.com/issues/121345407

```
Status: Won't Fix (Intended Behavior)
Please find the below comments from the development team.

"We are going to support the Saved Games in Play Games services, and implement the migration internally inside the Play Games services, so the API will no change for the developers.
Except, possibly, for the SCOPE_APPFOLDER, we are planing to introduce a copy of this constant in Play Games services, maybe with a with a different name, e.g. SCOPE_SAVEDGAMES.
To summarise: no migration required from the developers at the moment."
```

**Comment by olehkuznetsov** [comment]
This https://issuetracker.google.com/issues/121345407 is not related issue to 'sign-in and save games' one.

**Comment by EasybotGames** [comment]
so when it gonna fix in this week or next ?

**Comment by olehkuznetsov** [comment]
I will ping drive team today to check if there are any new updates.

**Comment by EasybotGames** [comment]
Update us as soon as they reply you and thanks for quick response.

**Comment by Vonak** [comment]
Status: Won't Fix (Intended Behavior)
Please find the below comments from the development team.

"We are going to support the Saved Games in Play Games services, and implement the migration internally inside the Play Games services, so the API will no change for the developers.
Except, possibly, for the SCOPE_APPFOLDER, we are planing to introduce a copy of this constant in Play Games services, maybe with a with a different name, e.g. SCOPE_SAVEDGAMES.
To summarise: no migration required from the developers at the moment."

**Comment by SCBbestof** [comment]
@Vonak  That task is unrelated, check the messages above. They have an internal tracking for this issue.

**Comment by EasybotGames** [comment]
ya that's what i am saying
i talked with @olehkuznetsov (one of the google official) he said he will ping drive team today and let us know how much more time it will take to be resolved.

**Comment by voxelbusters** [comment]
@olehkuznetsov Thanks for confirming.

**Comment by olehkuznetsov** [comment]
I've just got two news from the Drive team:
1) It seems full fix will not be available this week.
2) With the current fix state they were able to prepare the [workaround instruction](https://docs.google.com/document/d/1UkC1jhpvOr0au_FBCp3Ug2oNgTlNST1UM12tsCdWPhs).

**Comment by EasybotGames** [comment]
It is asking for permission,it is saying you will get email if your request is approved , can you please help with that .

**Comment by olehkuznetsov** [comment]
I've updated the [link](https://docs.google.com/document/d/1UkC1jhpvOr0au_FBCp3Ug2oNgTlNST1UM12tsCdWPhs) in previous comment - try again, please.

**Comment by EasybotGames** [comment]
Thanks i will check it out.

**Comment by EasybotGames** [comment]
i am stuck in configuration it is saying "At least one extension must be enabled and configured" what should i select and it is also asking for "Add-on extensions" what should i suppose to do with that ? because of that save changes button is inactive. Please help me out.

**Comment by olehkuznetsov** [comment]
Just enable 'Universal navigation extension' and put any valid link.

**Comment by EasybotGames** [comment]
what do you mean by valid link ,can i use my website link or privacy policy link.

**Comment by olehkuznetsov** [comment]
Yes, you have use your website link or probably some random one like http://asdfsayewrijsda.com should work too.

To quickly check if your project is fixed you can use the link:
https://developers.google.com/apis-explorer/#search/drive.apps.get/m/drive/v2/drive.apps.get
1. Put you APP ID to 'appid' field
2. Press Execute
3. Authorize if required
4. Check the Response code - if it is 200 your project is fixed.

appId |  
-- | --




appId |  
-- | --

**Comment by EasybotGames** [comment]
@olehkuznetsov  Thanks you very much for quick response mate appreciate your help so much.

**Comment by SCBbestof** [comment]
@olehkuznetsov Thanks for helping with this! 😃
I have two quick questions:
1. Any idea when the full fix will be done?
2. Will this workaround work after the fix or will we need to patch our apps to make them work again after?

**Comment by ProtoJerry** [comment]
At last it works. Thank you @olehkuznetsov for the update 👍 
![image](https://user-images.githubusercontent.com/44014255/53889287-d784f900-402e-11e9-8dd7-7d604f4535bd.png)

**Comment by rickyfreddy** [comment]
"You only need to fill in the required fields, and use placeholder information if you do not plan to publish the listing." How do I know if I need to publish the listing? Do I need to do that to make Google Play Saved Games work again?

**Comment by dannykay1** [comment]
I am also having the same problem with EnableSavedGames().  Everything works fine without it.

**Comment by manutyagi01** [comment]
I am also having the same issue but I am not using unity, I am using Libgdx + gdx-gamesvcs (Game Services Integration Library).....

**Comment by kaninhot004** [comment]
I can use it, Thanks for workaround guide! @olehkuznetsov

**Comment by bonedmatt** [comment]
As of today this is still not fixed.

**Comment by kaninhot004** [comment]
Do the workaround guide @bonedmatt

**Comment by wooyaggo** [comment]
I can't access the link.

https://docs.google.com/document/d/1UkC1jhpvOr0au_FBCp3Ug2oNgTlNST1UM12tsCdWPhs

What should I do?

**Comment by kaninhot004** [comment]
This link: https://drive.google.com/file/d/1Mh9JZKPiHr4iYO7JbmXGNX0O4RusQFz1/view

is okay for me.

**Comment by danieltanbear** [comment]
Link to google drive document does not work. Can a link to the pdf be provided directly?

**Comment by JesseLeeBusch** [comment]
Is there been a status update on the server side bug fix for this?

**Comment by kaninhot004** [comment]
![image](https://user-images.githubusercontent.com/9493525/54403077-55b55f80-4701-11e9-9b19-95291bd3a91a.png)

So 1 week passed, ETA is in 7-8 days later.

by the way you can workaround by his guides.

**Comment by M-allaham** [comment]
![تعليق توضيحي 2019-03-16 220135](https://user-images.githubusercontent.com/48628611/54480255-3c163400-4837-11e9-99b8-0277b932af1a.jpg)

what can i do please ?
is app id from google games or other id ?

**Comment by sptkjs** [comment]
Is full fix done? ETA is over.

**Comment by Sailendrakc** [comment]
Is this fully fixed yet??

**Comment by olehkuznetsov** [comment]
Should be fixed March 19th.
